packages/webpack-loader/src/css-loader.tsx

```diff
@@ -14,10 +14,13 @@ export default function CSSLoader(this: LoaderContext<void>): string {
  * Moves CSSloader to the end of the loader queue so it runs first.
  */
 export function pitch(this: LoaderContext<void>): void {
-  if (this.loaders[0].path !== __filename) {
+  if (this.loaders[0].pitch !== pitch) {
```

Comment:
Now the entry point filename + path isn't always the same so I had to look for another solution - I leaned on referential equality to do it instead.